Multisig.xcodeproj/project.pbxproj

```diff
@@ -113,6 +113,7 @@
 		0A29ECAA24C1EBFD00CE28FD /* Address+LocalizedStringKey.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0A29ECA924C1EBFD00CE28FD /* Address+LocalizedStringKey.swift */; };
 		0A339C5D24C1D16E00DAD84B /* Image+Blockies.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0A339C5C24C1D16E00DAD84B /* Image+Blockies.swift */; };
 		0A3F34DB2477E4E900E872AF /* KeyboardAdaptive.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0A3F34D82477E4E900E872AF /* KeyboardAdaptive.swift */; };
+		0A42C25424D0696D00ACA770 /* GetTransactionRequest.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0A42C25324D0696D00ACA770 /* GetTransactionRequest.swift */; };
```

Comment:
Totally good, but I prefer to call it TransactionDetailsRequest instead

Reply:
changed